app/views/projects/samples/clones/_errors.html.erb

```diff
@@ -0,0 +1,16 @@
+<div
+  data-controller="projects--samples--metadata--complete"
+  data-projects--samples--metadata--complete-filters-outlet=".filters"
+>
+  <p class="mb-4"><%= t(".description") %></p>
+
+  <% errors.each do |error| %>
+    <%= viral_alert(type:, message: error, classes: "mb-4") %>
```

Comment:
**suggestion:** I think this should just be one alert with a title like `The following errors occurred` (or something better :)), and then a list of each that failed.  It would take up a lot less vertical space.

Reply:
I just want the wording to be clear that no cloning occurred for validation errors and partial cloning occurs for sample errors. Screenshots with errors are updated. Please see 8a17ed77f999c15976b7de67eaeab8a6717c8957.